frontend/integration-tests/tests/cluster-settings.scenario.ts

```diff
@@ -0,0 +1,220 @@
+
+
+describe('Cluster Settings', () => {
+  const labels = {app: `${testName}-app`};
+  const name = `${testName}-cell`;
+  const text = `${name} Logs`;
+  beforeEach(async() => {
+    await browser.get(`${appHost}/settings/cluster/${testName}`);
+    await crudView.isLoaded();
+  });
+
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  it('display page title, horizontal navigation tab headings and pages', async() => {
+    expect(clusterSettingsView.heading.isPresent()).toBe(true);
+    await horizontalnavView.clickHorizontalTab('Overview');
+    await crudView.isLoaded();
+    await horizontalnavView.clickHorizontalTab('Cluster Operators');
+    await crudView.isLoaded();
+    await horizontalnavView.clickHorizontalTab('Global Configuration');
+    await crudView.isLoaded();
+  });
+  it('display overview channel update modal, update value and save, and check if selected value equal to updated value', async() => {
+    await horizontalnavView.clickHorizontalTab('Overview');
+    await crudView.isLoaded();
+    expect(clusterSettingsView.channelUpdateLink.isDisplayed()).toBe(true);
+    await clusterSettingsView.channelUpdateLink.click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.channelDropdownButton.isDisplayed()).toBe(true);
+    await clusterSettingsView.channelDropdownButton.click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.getSelectedChannel.get(0).isDisplayed()).toBe(true);
+    const selectedChannelValue = clusterSettingsView.getSelectedChannel.get(0).getText();
+    await clusterSettingsView.getSelectedChannel.get(0).click();
+
+    expect(clusterSettingsView.confirmActionButton.isDisplayed()).toBe(true);
+    await clusterSettingsView.confirmActionButton.click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.channelUpdateLink.getText()).toEqual(selectedChannelValue);
+    await crudView.isLoaded();
+  });
+
+  it('display overview channel update modal, and click cancel.', async() => {
+    await horizontalnavView.clickHorizontalTab('Overview');
+    await crudView.isLoaded();
+    expect(clusterSettingsView.channelUpdateLink.isDisplayed()).toBe(true);
+    await clusterSettingsView.channelUpdateLink.click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.channelPopupCancelButton.isDisplayed()).toBe(true);
+    await clusterSettingsView.channelPopupCancelButton.click();
+    await crudView.isLoaded();
+  });
+  it('display Cluster Operators page, click resource item link to display details, click yaml editor, update and save content. Check if updated values persist', async()=>{
+
+    await horizontalnavView.clickHorizontalTab('Cluster Operators');
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.resourceItemLink.get(4).isDisplayed()).toBe(true);
+    await clusterSettingsView.resourceItemLink.get(4).click();
+
+    await horizontalnavView.clickHorizontalTab('YAML');
```

Comment:
I would skip the YAML editor tests. Those are pretty well covered by CRUD. Just making sure the link works and details display is probably enough here. We might check something like the page header to know it's the right resource title.